library/src/image.cxx

```diff
@@ -25,19 +25,13 @@ class image::internals
 
   vtkSmartPointer<vtkImageImport> GetVTKImporter() const
   {
-    if (this->Buffer.size() != this->Width * this->Height * this->Channels)
```

Comment:
Why did you remove this block?

Reply:
Because this test just cannot be true. I could put an assert if you want.

Reply:
Yes please